app/router.js

```diff
@@ -56,6 +56,8 @@ Router.map(function () {
     this.route('release');
   });
 
+  this.route('browser-support');
```

Comment:
Could you place this line between the lines for `blog` and `community`? I'd like to keep the list alphabetized to help with maintenance. 🙂